Random fixes

rargulati opened the pull request:
This PR makes minor modifications to this package. Removing references to *ipfs where necessary, newlines (+/-), and using `fmt.Println` temporarily, where necessary (for informative debug when testing in `keep-network/keep-core`.

rargulati committed:
Remove reference sto go-ipfs and IpfsNode

The purpose of this package is to be independent of use in go-ipfs.

rargulati committed:
Remove and add newlines

Spacing for where it makes sense.

rargulati committed:
Use Println instead of log

For our purposes, temporarily use fmt.Println for an output message.

pdyraga commented on bootstrap.go:
```diff
@@ -79,14 +78,16 @@ func Bootstrap(
+		fmt.Println(
+			`no bootstrap nodes configured:
+                           may have difficulty connecting to the network"`,
+		)
```
It prints weird:
```
no bootstrap nodes configured:
                           may have difficulty connecting to the network"
```

Was that intentional?

pdyraga committed:
Print error about bootstrap peer connection failure

It's not the right way (TM) for logging but it's only temporary solution
before we do the logging in a right way in Keep.

pdyraga commented:
I added one `fmt.Printf` informing about the fact that bootstrap node connection failed. It's not the right approach for logging but it's only temporary before we do the logging in our own project in the right way.

pdyraga commented:
@dimpar Can you please take a look and if you agree with the changes here, merge?

dimpar reviewed (approved):
looks good
